specs/chrome_extensions.table

```diff
@@ -6,7 +6,8 @@ schema([
     Column("name", TEXT, "Extension display name"),
     Column("profile", TEXT, "The name of the Chrome profile that contains this extension"),
     Column("profile_path", TEXT, "The profile path"),
-    Column("identifier", TEXT, "Extension identifier (folder name)"),
+    Column("referenced_identifier", TEXT, "Extension identifier, as specified by the preferences file. Empty if the extension is not in the profile."),
+    Column("computed_identifier", TEXT, "Extension identifier, computed from its manifest. Empty in case of error.", aliases=["identifier"]),
```

Comment:
On reflection, I think we should call the computed identifier `identifier`. It matches how chrome computes it, and it matches our current usage.

Reply:
This makes perfect sense! I have updated the PR so that `identifier` is now the computed value